### DISCUSSION TITLE
Alphabet mode

### DISCUSSION THREAD

**Discussion by DarkPlayerr** [monkeytypegame/monkeytype#1833] [Ideas] [answered]
I feel like an alphabet mode would be nice in monkeytype. Saw the idea first on mkbhd's podcast when he mentioned typethealphabet.app which is fine but ugly so would like it with the monkeytype ui & theme.

**Comment by Miodec** [comment, marked as answer]
You can replicate this by using custom mode and stop on error: letter.

**Reply by DarkPlayerr** [reply]
i know about it, but the timer could be a bit more exact + show detailed results like type the alphabet does

**Reply by frogfrogfrogfrogfrog** [reply]
I think the information on Monketype would show enough if I'm being honest

but Miodec if your interested you could make a challenge out of the alphabet (such as typing it over 100 wpm or typing it 100 times)
